packages/data-api/scripts/dropAnalyticsMaterializedView.sql

```diff
@@ -0,0 +1,39 @@
+do $$
+declare
+  tStartTime TIMESTAMP;
+  source_table TEXT;
+  source_tables_array TEXT[] := array['answer', 'survey_response', 'entity', 'survey', 'question', 'data_source'];
+
+begin
+  RAISE NOTICE 'Dropping analytics materialized view...';
+  IF (SELECT EXISTS (
+    SELECT FROM pg_tables
+    WHERE schemaname = 'public'
+    AND tablename   = 'analytics'
+  ))
+  THEN
+    tStartTime := clock_timestamp();
+    PERFORM mv$removeMaterializedView('analytics', 'public');
+    RAISE NOTICE 'Dropped analytics table, took %', clock_timestamp() - tStartTime;
+  ELSE
+    RAISE NOTICE 'Analytics Materialized View does not exist, skipping';
+  END IF;
+
+  RAISE NOTICE 'Dropping Materialized View Logs...';
+  FOREACH source_table IN ARRAY source_tables_array LOOP
+    IF (SELECT EXISTS (
+      SELECT FROM pg_tables
+      WHERE schemaname = 'public'
+      AND tablename   = 'log$_' || source_table
+    ))
+    THEN
+      EXECUTE 'DROP TRIGGER IF EXISTS ' || source_table || '_trigger' || ' ON ' || source_table;
```

Comment:
Hmmm I know this is my fault since I wrote the original code you probably based yours off here, but I suppose we ought to be using disable/re-enable trigger here rather than drop/re-create?

Reply:
Aaah that's a great point. I should have thought of this! It would have even saved me the confusion of working out what these new triggers you were creating were...realised after a minute they were actually just the old triggers, and that you'd dropped them a couple of lines up, which made sense but will be clearer for future devs as a disable/re-enable pattern. I will do this now on both scripts!